app/views/dashboard.py

```diff
@@ -9,7 +9,7 @@
 from datetime import datetime
 
 from app.util import MobileSupportedView, most_recent_game
-from app.models import Game, Tag, Player, PlayerRole, SignupLocation, Legacy, SupplyCode
+from app.models import Game, Tag, Player, PlayerRole, SignupLocation, Legacy, SupplyCode, User
```

Comment:
Why do we import User here? Please remove it if it's not required.

Reply:
That was added to allow for checking the groups attached to an individual to see if they're in the legacy grouping. Lmk if there's another way to do this?